assets/stylesheets/screens/_dashboard.scss

```diff
@@ -1,6 +1,16 @@
 body.dashboard.index {
   .search-keyword {
-    @apply py-4 sm:py-8;
+    @apply relative py-4 sm:py-8;
```

Comment:
Should we use `@screen` directive to make it easy to read and maintain later?
https://tailwindcss.com/docs/functions-and-directives#screen
```CSS
.search-keyword {
  @apply relative py-4;

  @screen sm {
    @apply py-8;
  }
}
```

Reply:
It doesn't work for me 🤔
<img width="1385" alt="Screen Shot 2564-03-08 at 03 55 13" src="https://user-images.githubusercontent.com/749672/110254626-79df6a80-7fc2-11eb-9dc7-69db8fac2b96.png">